.github/workflows/deploy-screener.yml

```diff
@@ -0,0 +1,49 @@
+# This action deploys changes to the eligibility screener
+# Deploy steps:
+# - Build a docker release image and upload it to ECR and tag it.
+# - Apply the terraform changes (if applicable), pointing it to the docker image that was uploaded
+
+name: Elibility Screener Deploy
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        default_value: "test"
+        description: |
+          Choose an environment to deploy changes to.
+        type: choice
+        options:
+          - test
+          - stage
+  push:
+    branches:
+      - main
+      - alsia/add-cd # this is for testing purposes
+
+jobs:
+  build:
+    name: Build and Deploy Screener
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      # add secret key  + acct key to GH secrets
+      - name: Authenticate with AWS
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: arn:aws:iam::546642427916:role/deployment-action
+          role-duration-seconds: 3600
+
+      - name: Build and Tag Image
+        run: |
+          docker-compose build
```

Comment:
Should this be using `docker build` (and `--tag 546642427916.dkr.ecr.us-east-1.amazonaws.com/eligibility-screener-repo:latest`)?

I realize the command in the docker-compose file (`yarn dev`) is different from the default command in the Dockerfile (`yarn start`) and now I'm wondering if this actually builds a dev image instead of a release image 🤔